src/search/classic/search.cc

```diff
@@ -609,28 +609,11 @@ PositionHistory Search::GetPositionHistoryAtNode(const Node* node) const {
   return history;
 }
 
-namespace {
-std::vector<Move> GetNodeLegalMoves(const Node* node, const ChessBoard& board) {
-  if (!node) return {};
-  std::vector<Move> moves;
-  if (node && node->HasChildren()) {
-    moves.reserve(node->GetNumEdges());
-    std::transform(node->Edges().begin(), node->Edges().end(),
-                   std::back_inserter(moves),
-                   [](const auto& edge) { return edge.GetMove(); });
-    return moves;
-  }
-  return board.GenerateLegalMoves();
-}
-}  // namespace
-
 std::optional<EvalResult> Search::GetCachedNNEval(const Node* node) const {
```

Comment:
nit, optional: maybe rename the function so something like `GetCacheNNValueHead` to make it clear that it doesn't return policy now.

Reply:
What about removing the function completely?

Reply:
Yeah even better.